src/fiatconnect/KycLanding.test.tsx

```diff
@@ -65,31 +64,60 @@ describe('KycLanding', () => {
       fireEvent.press(getByTestId('checkbox'))
       expect(getByTestId('PersonaButton')).not.toBeDisabled()
     })
-    it('passes a callback function for successful persona result', () => {
-      const mockPersonaButton = jest.fn(() => <View></View>)
-      jest.spyOn(Persona, 'default').mockImplementation(mockPersonaButton)
-      render(
+    it('triggers analytics and dispatches persona started when persona button is pressed', () => {
+      ValoraAnalytics.track = jest.fn()
+      const { getByTestId } = render(
         <Provider store={store}>
           <KycLanding {...props} />
         </Provider>
       )
-      expect(Persona.default).toHaveBeenCalledWith(
-        expect.objectContaining({ onSuccess: expect.any(Function) }), // props for Persona
-        expect.anything()
+      fireEvent.press(getByTestId('checkbox'))
+      fireEvent.press(getByTestId('PersonaButton'))
+      expect(ValoraAnalytics.track).toHaveBeenCalledWith(CICOEvents.persona_kyc_start)
+      expect(store.dispatch).toHaveBeenCalledWith(personaStarted())
+    })
+    it('dispatches post kyc when persona succeeds', () => {
+      const { getByTestId } = render(
+        <Provider store={store}>
+          <KycLanding {...props} />
+        </Provider>
       )
-      jest.restoreAllMocks()
+      fireEvent.press(getByTestId('checkbox'))
+      fireEvent.press(getByTestId('PersonaSuccessButton'))
```

Comment:
Could you add a comment to each of these explaining? Its kind of hacky and took me a bit to understand what was going on
```suggestion
      // 'PersonaSuccessButton' is a mock that calls onSuccess when pressed
      fireEvent.press(getByTestId('PersonaSuccessButton'))
```